src/main/java/com/nirima/jenkins/plugins/docker/action/DockerBuildImageAction.java

```diff
@@ -29,27 +29,38 @@
     public final boolean cleanupWithJenkinsJobDelete;
     public final boolean pushOnSuccess;
 
+    public final boolean noCache;
+    public final boolean pull;
+
     @Deprecated
     public DockerBuildImageAction(String containerHost,
                                   String containerId,
                                   String taggedId,
                                   boolean cleanupWithJenkinsJobDelete,
-                                  boolean pushOnSuccess) {
+                                  boolean pushOnSuccess,
+                                  final boolean noCache,
```

Comment:
Code style: "Try to fit in" - don't use `final` here.
While, personally, I'm a big fan of `finally` and use it every time I can, that hasn't been done in the rest of this class so, unless you want to first file a PR that reformats/re-styles this code to use `final` everywhere, please ensure that your new additions use the same code-style as is prevalent in the rest of this code.
See the "Coding style" section in the [contribution guidelines](https://github.com/jenkinsci/docker-plugin/blob/master/CONTRIBUTING.md) for more details.